merge_sort.rb

```diff
@@ -0,0 +1,39 @@
+
+def merge(left_arr, right_arr)
+  new_arr = []
+  if left_arr != nil && right_arr != nil
```

Comment:
replace `if !` by `unless` and `== nil` by `.nil?`
`unless left_arr.nil? || right_arr.nil?`